build.gradle

```
@@ -277,6 +283,25 @@ task tag(type: Exec) {
 	}
 }
 
+mainClassName = "dev.jbang.cli.Jbang"
+
+task generateManpageAsciiDoc(type: JavaExec) {
+	dependsOn(classes)
+	group = "Documentation"
+	description = "Generate AsciiDoc manpage"
+	classpath(configurations.compile, configurations.annotationProcessor, sourceSets.main.runtimeClasspath)
+	main 'picocli.codegen.docgen.manpage.ManPageGenerator'
+	args mainClassName, "--outdir=${buildDir}/generated-picocli-docs"
+}
+
+asciidoctor {
+	dependsOn(generateManpageAsciiDoc)
+	sourceDir = file("${projectDir}/misc/doc-templates")
+	outputDir = file("${buildDir}/docs")
+	logDocuments = true
+	backends 'manpage'
```

```suggestion
	backends = ['html5', 'manpage']
```

then we also get the html5 version.

---

I removed html5 because the result was not totally ok: many pages, no links between them.